json_str/json_str.h

```diff
@@ -26,6 +26,7 @@ typedef struct json_command
   unsigned int duration;
   int speed;
   MotorDirection direction;
+  uint8_t mode;

```

Comment:
Maybe `bool mode;`?

Reply:
Refer to issue [#6](https://github.com/IoT-project-groups-1-2/g1-map-robot/issues/6#issuecomment-1312735363)
We will have to change json message in the future anyway, so it can be changed to bool later, if you are ok with it.

Reply:
We will forget :D